diff --git a/files/kind.yaml b/files/kind.yaml
index c67069c..bbdb345 100644
--- a/files/kind.yaml
+++ b/files/kind.yaml
@@ -56,7 +56,16 @@ nodes:
       - containerPort: 50000
         hostPort: 50000
         listenAddress: "0.0.0.0"
+    extraMounts:
+      - containerPath: /var/lib/kubelet/config.json
+        hostPath: /home/opnfv/.docker/config.json
   - role: worker
     image: kindest/node:v1.25.3
+    extraMounts:
+      - containerPath: /var/lib/kubelet/config.json
+        hostPath: /home/opnfv/.docker/config.json
   - role: worker
     image: kindest/node:v1.25.3
+    extraMounts:
+      - containerPath: /var/lib/kubelet/config.json
+        hostPath: /home/opnfv/.docker/config.json
diff --git a/tests/files/kind.yaml b/tests/files/kind.yaml
index c67069c..bbdb345 100644
--- a/tests/files/kind.yaml
+++ b/tests/files/kind.yaml
@@ -56,7 +56,16 @@ nodes:
       - containerPort: 50000
         hostPort: 50000
         listenAddress: "0.0.0.0"
+    extraMounts:
+      - containerPath: /var/lib/kubelet/config.json
+        hostPath: /home/opnfv/.docker/config.json
   - role: worker
     image: kindest/node:v1.25.3
+    extraMounts:
+      - containerPath: /var/lib/kubelet/config.json
+        hostPath: /home/opnfv/.docker/config.json
   - role: worker
     image: kindest/node:v1.25.3
+    extraMounts:
+      - containerPath: /var/lib/kubelet/config.json
+        hostPath: /home/opnfv/.docker/config.json
diff --git a/tests/templates/jenkins_kind.yaml.j2 b/tests/templates/jenkins_kind.yaml.j2
index 4fe6414..749697e 100644
--- a/tests/templates/jenkins_kind.yaml.j2
+++ b/tests/templates/jenkins_kind.yaml.j2
@@ -15,10 +15,19 @@
           nodes:
             - role: control-plane
               image: kindest/node:v1.25.3
+              extraMounts:
+                - containerPath: /var/lib/kubelet/config.json
+                  hostPath: /home/opnfv/.docker/config.json
             - role: worker
               image: kindest/node:v1.25.3
+              extraMounts:
+                - containerPath: /var/lib/kubelet/config.json
+                  hostPath: /home/opnfv/.docker/config.json
             - role: worker
               image: kindest/node:v1.25.3
+              extraMounts:
+                - containerPath: /var/lib/kubelet/config.json
+                  hostPath: /home/opnfv/.docker/config.json
           EOF
           curl -Lo kind https://kind.sigs.k8s.io/dl/v0.17.0/kind-linux-amd64
           chmod +x kind
diff --git a/tests/templates/kind.yaml.j2 b/tests/templates/kind.yaml.j2
index 463558b..25ca988 100644
--- a/tests/templates/kind.yaml.j2
+++ b/tests/templates/kind.yaml.j2
@@ -9,7 +9,16 @@ containerdConfigPatches:
 nodes:
 - role: control-plane
   image: kindest/node:v1.25.3
+  extraMounts:
+    - containerPath: /var/lib/kubelet/config.json
+      hostPath: /home/opnfv/.docker/config.json
 - role: worker
   image: kindest/node:v1.25.3
+  extraMounts:
+    - containerPath: /var/lib/kubelet/config.json
+      hostPath: /home/opnfv/.docker/config.json
 - role: worker
   image: kindest/node:v1.25.3
+  extraMounts:
+    - containerPath: /var/lib/kubelet/config.json
+      hostPath: /home/opnfv/.docker/config.json
