diff --git a/files/kind.yaml b/files/kind.yaml
index 8e3bf5a..c959739 100644
--- a/files/kind.yaml
+++ b/files/kind.yaml
@@ -52,7 +52,16 @@ nodes:
       - containerPort: 50000
         hostPort: 50000
         listenAddress: "0.0.0.0"
+    extraMounts:
+      - containerPath: /var/lib/kubelet/config.json
+        hostPath: /home/opnfv/.docker/config.json
   - role: worker
     image: kindest/node:v1.25.3
+    extraMounts:
+      - containerPath: /var/lib/kubelet/config.json
+        hostPath: /home/opnfv/.docker/config.json
   - role: worker
     image: kindest/node:v1.25.3
+    extraMounts:
+      - containerPath: /var/lib/kubelet/config.json
+        hostPath: /home/opnfv/.docker/config.json
diff --git a/tests/files/kind.yaml b/tests/files/kind.yaml
index 8e3bf5a..c959739 100644
--- a/tests/files/kind.yaml
+++ b/tests/files/kind.yaml
@@ -52,7 +52,16 @@ nodes:
       - containerPort: 50000
         hostPort: 50000
         listenAddress: "0.0.0.0"
+    extraMounts:
+      - containerPath: /var/lib/kubelet/config.json
+        hostPath: /home/opnfv/.docker/config.json
   - role: worker
     image: kindest/node:v1.25.3
+    extraMounts:
+      - containerPath: /var/lib/kubelet/config.json
+        hostPath: /home/opnfv/.docker/config.json
   - role: worker
     image: kindest/node:v1.25.3
+    extraMounts:
+      - containerPath: /var/lib/kubelet/config.json
+        hostPath: /home/opnfv/.docker/config.json
diff --git a/tests/templates/jenkins_kind.yaml.j2 b/tests/templates/jenkins_kind.yaml.j2
index a25357d..9b01eaa 100644
--- a/tests/templates/jenkins_kind.yaml.j2
+++ b/tests/templates/jenkins_kind.yaml.j2
@@ -11,10 +11,19 @@
           nodes:
             - role: control-plane
               image: kindest/node:v1.25.3
+              extraMounts:
+                - containerPath: /var/lib/kubelet/config.json
+                  hostPath: /home/opnfv/.docker/config.json
             - role: worker
               image: kindest/node:v1.25.3
+              extraMounts:
+                - containerPath: /var/lib/kubelet/config.json
+                  hostPath: /home/opnfv/.docker/config.json
             - role: worker
               image: kindest/node:v1.25.3
+              extraMounts:
+                - containerPath: /var/lib/kubelet/config.json
+                  hostPath: /home/opnfv/.docker/config.json
           EOF
           curl -Lo kind https://kind.sigs.k8s.io/dl/v0.17.0/kind-linux-amd64
           chmod +x kind
diff --git a/tests/templates/kind.yaml.j2 b/tests/templates/kind.yaml.j2
index 15cdc5d..be5e641 100644
--- a/tests/templates/kind.yaml.j2
+++ b/tests/templates/kind.yaml.j2
@@ -5,7 +5,16 @@ networking:
 nodes:
 - role: control-plane
   image: kindest/node:v1.25.3
+  extraMounts:
+    - containerPath: /var/lib/kubelet/config.json
+      hostPath: /home/opnfv/.docker/config.json
 - role: worker
   image: kindest/node:v1.25.3
+  extraMounts:
+    - containerPath: /var/lib/kubelet/config.json
+      hostPath: /home/opnfv/.docker/config.json
 - role: worker
   image: kindest/node:v1.25.3
+  extraMounts:
+    - containerPath: /var/lib/kubelet/config.json
+      hostPath: /home/opnfv/.docker/config.json
